auth-service/src/main/kotlin/uk/gov/communities/delta/auth/controllers/internal/AdminGetUserController.kt

```diff
@@ -32,22 +29,14 @@ class AdminGetUserController(
 
         val cn = call.request.queryParameters["userCn"]!!
         logger.atInfo().log("Getting info for user $cn")
-        val user: LdapUser
+        val user: UserLookupService.UserWithRoles
         try {
-            user = userLookupService.lookupUserByCn(cn)
+            user = userLookupService.lookupUserByCNAndLoadRoles(cn)
         } catch (e: NameNotFoundException) {
+            logger.warn("User not found $cn")
             return call.respond(HttpStatusCode.NotFound, "User not found")
         }
-        coroutineScope {
-            val allOrganisations = async { organisationService.findAllNamesAndCodes() }
-            val allAccessGroups = async { accessGroupsService.getAllAccessGroups() }
-
-            val roles = memberOfToDeltaRolesMapperFactory(
-                user.cn, allOrganisations.await(), allAccessGroups.await()
-            ).map(user.memberOfCNs)
-            call.respond(UserWithRoles(user, roles))
-        }
-
+        call.respond(UserWithRoles(user.user, user.roles))
```

Comment:
Isn't this just `call.respond(user)`?

Reply:
They're different classes with the same name, identical except this on is `@Serializable`, I'll commonise them